src/pip/_internal/download.py

```diff
@@ -72,6 +72,18 @@
 logger = logging.getLogger(__name__)
 
 
+def looks_like_ci():
+    # type: () -> bool
+    """
+    Return whether it looks like pip is running under CI.
+    """
+    return (
+        'BUILD_BUILDID' in os.environ or
```

Comment:
seems kinda repetitive, maybe instead ``any(key in os.environ for key in ('A', 'B))``